provider/workspace_owner.go

```diff
@@ -53,7 +53,16 @@ func workspaceOwnerDataSource() *schema.Resource {
 			_ = rd.Set("session_token", os.Getenv("CODER_WORKSPACE_OWNER_SESSION_TOKEN"))
 			_ = rd.Set("oidc_access_token", os.Getenv("CODER_WORKSPACE_OWNER_OIDC_ACCESS_TOKEN"))
 
-			return nil
+			if os.Getenv("CODER_WORKSPACE_OWNER_LOGIN_TYPE") == "" {
+				diags := req.Config.Get(ctx, &rd)
```

Comment:
There is no `req` variable in scope here, but I don't think this line is necessary (see comment on next line).